src/common/Hero.ts

```diff
@@ -3,15 +3,19 @@ export interface HeroHeadingProps {
   subtitle?: string
 }
 
-export interface HeroButtonProps {
+export interface HeroButtonsProps {
```

Comment:
What is this common folder? Just to export props? Are the props common for all templates (classic + next)?

Reply:
Ideally yes, components with the same functionality should share the same props to allow for easy switching between templates.

Reply:
okay sgtm